cypress/component/FreeformInputWidget.cy.tsx

```diff
@@ -0,0 +1,366 @@
+import React from 'react'
+
+import { FreeformInputWidget, TooltipProvider } from '../../src'
```

Comment:
I think we need to make sure this widget works also when it's inside an `ExclusiveGroupWidget`.  See https://github.com/ecmwf-projects/cads-ui-library/blob/main/cypress/component/ExclusiveGroupWidget.cy.tsx for examples.

Reply:
Also, https://github.com/ecmwf-projects/cads-ui-library/blob/main/src/utils/widgetFactory.tsx

Reply:
Added at https://github.com/ecmwf-projects/cads-ui-library/pull/48/commits/667e176e034e7babb0e181a3a60af87ac4a16252 .